spec/compiler/normalize/op_assign_spec.cr

```diff
@@ -1,8 +1,10 @@
 require "../../spec_helper"
 
 describe "Normalize: op assign" do
-  it "normalizes var +=" do
-    assert_normalize "a = 1; a += 2", "a = 1\na = a + 2"
+  ["+", "-", "*", "&+", "&-", "&*"].each do |op|
```

Comment:
seem like in the rest of this file each test case is broken out.  I think it might make it easier to understand if they are different test cases.